app/services/stormpath/rails/controller_authentication/from_bearer_auth.rb

```diff
@@ -1,34 +0,0 @@
-module Stormpath
-  module Rails
-    class ControllerAuthentication
-      class FromBearerAuth
-        attr_reader :authorization_header
-
-        RESCUE_CLASSES = [
-          Stormpath::Oauth::Error,
-          JWT::DecodeError,
-          AccountFromAccessToken::AuthenticationWithRefreshTokenAttemptError,
-          AccountFromAccessToken::DifferentIssuerError
-        ].freeze
-
-        def initialize(authorization_header)
-          @authorization_header = authorization_header
-        end
-
-        def authenticate!
-          begin
-            AccountFromAccessToken.new(bearer_access_token).account
-          rescue *RESCUE_CLASSES
```

Comment:
Create a request spec called 'authentication_spec' where you ping the me route and you check to see if the errors raised from ControllerAuthentication get saved in the Controller. Failing specs first.